build-system/runner/test/org/ampproject/AmpPassTest.java

```diff
@@ -17,7 +17,7 @@
       "dev.fine");
 
   @Override protected CompilerPass getProcessor(Compiler compiler) {
-    return new AmpPass(compiler, suffixTypes);
+    return new AmpPass(compiler, true, suffixTypes);
```

Comment:
sorry, did you mean to do something like `return new AmpPass(compiler, /* isProd */ true, suffixTypes);` ?